Firmware/first_lay_cal.cpp

```diff
@@ -163,42 +170,49 @@ void lay1cal_before_meander()
 //! @brief Print meander start
 void lay1cal_meander_start(float layer_height, float extrusion_width)
 {
-    enquecommand_P(PSTR("G1 X50 Y155"));
-
-    static const char fmt1[] PROGMEM = "G1 Z%-.3f F7200";
+#ifndef NEW_FIRST_LAYER_CAL
+    enquecommand_P(PSTR("G1X50Y155"));
+#endif //_NEW_FIRST_LAYER_CAL
+    static const char fmt1[] PROGMEM = "G1Z%gF7200";
     enquecommandf_P(fmt1, layer_height);
-
-    enquecommand_P(PSTR("G1 F1080"));
-
-    enquecommandf_P(extrude_fmt,  75, 155, count_e(layer_height, extrusion_width * 4.f, 25));
-    enquecommandf_P(extrude_fmt, 100, 155, count_e(layer_height, extrusion_width * 2.f, 25));
-    enquecommandf_P(extrude_fmt, 200, 155, count_e(layer_height, extrusion_width, 100));
-    enquecommandf_P(extrude_fmt, 200, 135, count_e(layer_height, extrusion_width, 20));
+    enquecommand_P(PSTR("G1F1080"));
+#ifdef NEW_FIRST_LAYER_CAL
+    enquecommandf_P(extrude_fmt_Y, short_length, count_e(layer_height, extrusion_width, short_length));
+#endif //_NEW_FIRST_LAYER_CAL
+
+    enquecommand_P(PSTR("G91"));
```

Comment:
maybe extract this to messages.h/cpp?

Reply:
Also, shouldn't this line be above the first meander move for the new calibration? This feels slightly misplaced.

Reply:
Fixed @3d-gussner